src/components/RoasterRegisterContainer.js

```diff
@@ -41,7 +46,11 @@ class RoasterRegisterContainer extends Component {
             }
         };
 
-        dispatch(createRoaster(data));
+        dispatch(createRoaster(data)).then(() => {
```

Comment:
this probably works, but i've run into a couple problems relying on the promise returned from a dispatch call. might be worth using `componentWillReceiveProps` and looking at the props object to see if the user is created.